ipy2to3_other.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
Right, but having the file called ipy2to3_other.py is confusing to people who want to use the tool. It is better to have one file that then is smart enough to decide what code it should be using from the ipython version.